examples/react/src/App.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import {Router} from 'director';
+
+import TodoFooter from './Footer';
+import TodoItem from './TodoItem';
+import {ALL_TODOS, ACTIVE_TODOS, COMPLETED_TODOS} from './utils'
+
+const ENTER_KEY = 13;
+
+class App extends Component {
+  state = {
+      nowShowing: ALL_TODOS,
+      editing: null,
+      newTodo: ''
+  }
+  componentDidMount () {
+    var setState = this.setState;
+    var router = Router({
+      '/': setState.bind(this, {nowShowing: ALL_TODOS}),
+      '/active': setState.bind(this, {nowShowing: ACTIVE_TODOS}),
+      '/completed': setState.bind(this, {nowShowing: COMPLETED_TODOS})
+    });
+    router.init('/');
+  }
+  handleChange (event) {
+    this.setState({newTodo: event.target.value});
+  }
+  handleNewTodoKeyDown (event) {
+    if (event.keyCode !== ENTER_KEY) {
+      return;
+    }
+
+    event.preventDefault();
+
+    var val = this.state.newTodo.trim();
```

Comment:
Please use `const` whenever possible, and fall back to `let`. Please don’t use `var`.